runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -221,10 +222,12 @@ private void drawSkillImage(Graphics2D graphics, XpGlobe xpGlobe, int x, int y)
 			return;
 		}
 
+		BufferedImage resized = ImageUtil.resizeImage(skillImage, config.xpOrbSize() / 2, config.xpOrbSize() / 2);
```

Comment:
I don't think this should be resizing the image every single render cycle

Reply:
To expand on this, you can generate the image once in the plugin, and update it on config change accordingly, then get that image from the overlay so it is not generated once per frame.